pkg/drivers/kic/oci/oci.go

```diff
@@ -63,6 +61,11 @@ func CreateContainerNode(p CreateParams) error {
 		// label the node with the role ID
 		"--label", fmt.Sprintf("%s=%s", nodeRoleKey, p.Role),
 	}
+	// Podman does not allow setting resources when cgroup is specified
+	// exact podman error: "Error: invalid configuration, cannot set resources with rootless containers not using cgroups v2 unified mode"
+	if p.OCIBinary != Podman {
+		runArgs = append(runArgs, fmt.Sprintf("--cpus=%s", p.CPUs), fmt.Sprintf("--memory=%s", p.Memory))
```

Comment:
If possible, we should emit a warning whenever the users preferences are ignored.

Reply:
done